plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialAttached.java

```diff
@@ -129,6 +147,10 @@ public void adjust(Mechanism mechanism) {
         // For a lantern, this sets whether it is hanging from the ceiling.
         // For a gate, this sets whether it is lowered to attach to a wall block.
         // For a mangrove_propagule, this sets whether it is hanging from the block above it.
+        // For a tripwire, this sets whether a tripwire hook or string forms a complete tripwire circuit and is ready to trigger.
+        // Updating the property on a tripwire hook will change the texture to indicate a connected string, but will not have
+        // any effect when used on the tripwire string itself. It may however still be used to check whether the string forms a circuit.

```

Comment:
These lines are split really awkwardly. If you must insert a newline, place it after a period or a comma, not just in the middle.